cpp/frontend_pyreadline.cc

```diff
@@ -0,0 +1,82 @@
+#include "frontend_pyreadline.h"
+
+#include <assert.h>
+
+#include "_build/detected-config.h"
```

Comment:
This header file also has a lot of "OVM" stuff to get rid of, but that's OK for now